src/index.js

```diff
@@ -39,7 +39,8 @@ class ReactIntlUniversal {
       urlLocaleKey: null, // URL's query Key to determine locale. Example: if URL=http://localhost?lang=en-US, then set it 'lang'
       cookieLocaleKey: null, // Cookie's Key to determine locale. Example: if cookie=lang:en-US, then set it 'lang'
       locales: {}, // app locale data like {"en-US":{"key1":"value1"},"zh-CN":{"key1":"值1"}}
-      warningHandler: console.warn // ability to accumulate missing messages using third party services like Sentry
+      warningHandler: console.warn, // ability to accumulate missing messages using third party services like Sentry
+      escapeHtml: true // disable escape html in variable mode
```

Comment:
Surely this should default to `false`?
```suggestion
      escapeHtml: false // disable escape html in variable mode
```